src/axolotl/prompt_strategies/llama2_chat.py

```diff
@@ -81,8 +81,9 @@ class LLama2ChatTokenizingStrategy(PromptTokenizingStrategy):
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self.sequence_len = 4096
-        self.tokenizer.add_special_tokens({"pad_token": "<pad>"})
+        self.tokenizer.add_special_tokens(
+            {"pad_token": getattr(self, "pad_token", "<pad>")}
```

Comment:
I'm not sure there's reason to `getattr` as the class nor base class has this attribute.

Reply:
When I step through this with a debugger the `pad_token` attribute is indeed present as long as you supply it in the config

Reply:
Oh I see. Are you sure, it wasn't `self.tokenizer.pad_token`? I couldn't find the declaration of that attribute

Reply:
🤦🏽  yes so sorry that is correct!!!  I'm so sorry making the change.  Argghhh can't believe I didn't see that

Reply:
no worries :)